_posts/2017-10-09-omero-5-4-0.md

```diff
@@ -0,0 +1,87 @@
+   populate_metadata script
+-  OMERO.web UI fixes:
+   * improved display of Plates and Wells
+   * fixed label position for Wells
+   * added the ability to display Image and Well metadata in the Tables 
+     section for the same Plate
+   * improved copy/paste of rendering settings workflow
+   * improved layout in left-hand panel
+   * fixed filtering issue
+   * added support for administrators with restricted privileges to create
+     Project/Dataset for other users
+   * rolled back the display of tables in the viewer
+
+Sysadmin changes include:
+
+-  added support for the creation of administrators with restricted privileges
+   in OMERO.web admin panel
+-  added method to create administrators with restricted privileges specifying
+   a password
+-  added specific installation instructions for Debian 9
+-  added configuration to limit queries that public users can do in OMERO.web
+-  created minimal NGINX configuration file that can be included in a fixed
+   file to allow custom NGINX options to be defined only once (e.g. SSL
+   options)
+-  installed django-redis by default
+-  CLI improvements and fixes:
+   * fixed admin plugin so ``cleanse`` can handle larger directories
+   * added to chown plugin ability to target all of given users' data
+   * adjusted handling of standard input 
+   * added infrastructure to load external CLI plugins
+   * dropped support for command ``admin ports``
+
+Developer updates include:
+
+-  added method to JavaGateway to manipulate admin privileges
+-  fixed issue with JSONP decorator
+-  removed SciPy dependency
+-  adjusted OMERO.blitz API to allow some query results to be cached
+-  added support to the Rendering engine to update a series of settings in one
+   call
+-  added method to OMERO.py to manipulate advanced rendering settings
+-  allowed the maven repository to be overridden
+-  removed unused 3rd party libraries in OMERO.web
+-  added support for PyTables version 3.4+
+-  deprecated Path Object in OMERO model
+-  updated the documentation for server installation on Mac OS to no longer
+   use the homebrew formulae from https://github.com/ome/homebrew-alt (these
+   are not working and will not be fixed)
+
+Further changes to the Python BlitzGateway are described in [What's new for OMERO 5.4.0 for developers](https://docs.openmicroscopy.org/omero/5.4.0/developers/whatsnew.html).
+
+This release also upgrades the version of Bio-Formats which OMERO uses to [5.7.1](https://docs.openmicroscopy.org/bio-formats/5.7.1/about/whats-new.html).
+
+Upgrade information is on the [server upgrade page](https://docs.openmicroscopy.org/omero/5.4.0/sysadmins/server-upgrade.html). 
+Sysadmins who used the workaround for the [latest security advisory]({{ site.baseurl }}/security/advisories/2017-SV4-guest-user) rather than
+upgrading to 5.3.5 will also find instructions for reversing it should they
+wish to, as this release also includes the fix.
```

Comment:
This is a bit indefinite and ambiguous.  If you applied the fix, do you need to remove it, or not?

Reply:
Depends if you want to restore guest access I guess.

Reply:
This is now up in the air anyway given the further issues discovered today

Reply:
OK.  I think the reasons one might want to back out the fix should be clearly stated here, since that's not at all obvious to me reading it!